src/features/clean.ts

```diff
@@ -13,7 +14,7 @@ export async function cleanOutDir(configs: ResolvedOptions[]): Promise<void> {
 
   for (const config of configs) {
     if (!config.clean.length) continue
-    const files = await glob(config.clean, {
+    const files = await glob(config.clean.map(slash), {
```

Comment:
It should be normalized in `resolveClean` fn, instead of here.

Reply:
Updated.